addons/point_of_sale/controllers/main.py

```diff
@@ -133,10 +133,17 @@ def invoice_request_screen(self, **kwargs):
                 else:
                     errors['generic'] = _("No sale order found.")
 
+        fetch_order_from_qr = request.env['pos.order'].sudo().search([('uuid', '=', order_uuid)], limit=1)
+        if not fetch_order_from_qr:
+            return {"error": "Order not found"}
```

Comment:
Need to check the condition here 
i.e. when the `order_uuid` has the value then only perform the search operations!